README.md

```diff
@@ -1,7 +1,46 @@
-# babel_fish
+# MerC
 
 # A Python tool for detecting and translating easy to translate C macros
```

Comment:
Drop the leading # since this text is not part of the document title

```suggestion
A Python tool for detecting and translating easy to translate C macros
```